### PR TITLE
started the game logic by building out a print menu and validator, st…

### PR DESCRIPTION
**Pull request by howardjian**
…ill in progress

### CODE REVIEW

File: src/com/nguyeningcode/blackjack/BlackJackUtil.java [hunk 1]
```diff
@@ -119,4 +121,93 @@ public static boolean continueToGameLogic() {
...
+                printMenu(hand);
+            }
+
+        } while (!validateMenuInput(hand, menuOption) || error);
+        switch(menuOption){
+            case 1:
+                hand.playable = false;
+                break;
+            case 2:
+                hand.addCard()
```
Comment (jurytan) [inline src/com/nguyeningcode/blackjack/BlackJackUtil.java]: add a semicolon, then add a break

File: src/com/nguyeningcode/blackjack/BlackJackUtil.java [hunk 2]
```diff
@@ -119,4 +121,93 @@ public static boolean continueToGameLogic() {
...
+
+    public static void playerOptions(Player player, Hand hand) {
+        System.out.print(player.getName() + ", your hand is: " + hand.toString());
+        printMenu(hand);
+
+        boolean error;
+        int menuOption = 0;
+
+        do {
+            try {
```
Comment (jurytan) [inline src/com/nguyeningcode/blackjack/BlackJackUtil.java:141]: add your printMenu(hand) here

File: src/com/nguyeningcode/blackjack/BlackJackUtil.java [hunk 3]
```diff
@@ -119,4 +121,93 @@ public static boolean continueToGameLogic() {
...
+                error = false;
+                menuOption = Integer.parseInt(in.nextLine())
+            } catch(Exception e) {
+                System.out.println("Your selection of '" + menuOption + "' is invalid")
+                printMenu(hand);
```
Comment (jurytan) [inline src/com/nguyeningcode/blackjack/BlackJackUtil.java]: Remove this

File: src/com/nguyeningcode/blackjack/BlackJackUtil.java [hunk 4]
```diff
@@ -119,4 +121,93 @@ public static boolean continueToGameLogic() {
...
+            }
+
+            if(!validateMenuInput(hand, menuOption) && !error) {
+                System.out.println("You selected an invalid option");
+                printMenu(hand);
```
Comment (jurytan) [inline src/com/nguyeningcode/blackjack/BlackJackUtil.java]: Remove this

File: src/com/nguyeningcode/blackjack/BlackJackUtil.java [hunk 5]
```diff
@@ -119,4 +121,93 @@ public static boolean continueToGameLogic() {
...
+                if(hand.playable) {
+                    playerOptions(player, hand);
+                }
+            }
+        }
+    }
+
+    public static void playerOptions(Player player, Hand hand) {
+        System.out.print(player.getName() + ", your hand is: " + hand.toString());
+        printMenu(hand);
```
Comment (jurytan) [inline src/com/nguyeningcode/blackjack/BlackJackUtil.java]: Remove this

File: src/com/nguyeningcode/blackjack/BlackJackUtil.java [hunk 6]
```diff
@@ -119,4 +121,93 @@ public static boolean continueToGameLogic() {
...
+
+            if(!validateMenuInput(hand, menuOption) && !error) {
+                System.out.println("You selected an invalid option");
+                printMenu(hand);
+            }
+
+        } while (!validateMenuInput(hand, menuOption) || error);
+        switch(menuOption){
+            case 1:
+                hand.playable = false;
```
Comment (jurytan) [inline src/com/nguyeningcode/blackjack/BlackJackUtil.java]: hand.setPlayable(false)

File: src/com/nguyeningcode/blackjack/BlackJackUtil.java [hunk 7]
```diff
@@ -119,4 +121,93 @@ public static boolean continueToGameLogic() {
         return false;
     }
 
+    public static void roundLogic() {
+        for(Player player : Game.players) {
+            for(Hand hand : player.hand) {
```
Comment (jurytan) [inline src/com/nguyeningcode/blackjack/BlackJackUtil.java]: player.hands

File: src/com/nguyeningcode/blackjack/BlackJackUtil.java (hunk 2 above)
Comment (howardjian) [reply src/com/nguyeningcode/blackjack/BlackJackUtil.java:141]: finished

**Pull request merged**